### DISCUSSION TITLE
How to export header when installing my project?

### DISCUSSION THREAD

**Discussion by Siudya** [xmake-io/xmake#5112] [Q&A] [answered]
Repo: https://github.com/Siudya/saifcpp
Branch: main

I try `xmake && xmake install -o install_dir`
But headers in include are not exported to `install_dir/include`. What declaration should I add to `xmake.lua` to export headers?

**Comment by star-hengxing** [comment, marked as answer]
https://xmake.io/#/manual/project_target?id=targetadd_headerfiles

**Reply by Siudya** [reply]
Thanks, very helpful
